java-rest-service/accelerator.yaml

```diff
@@ -317,6 +317,10 @@ engine:
           regex:
             pattern: "springdoc-openapi-webmvc-core"
             with: "'springdoc-openapi-starter-webmvc-api'"
+        - type: ReplaceText
```

Comment:
Why is this needed, AFAICT we don't specify the snakeyaml version

Reply:
This PR adds the snakeyaml version to fix a CVE.